lib/mongo_ecto.ex

```diff
@@ -758,7 +758,13 @@ defmodule Mongo.Ecto do
   @list_collections_query ["$and": [[name: ["$not": special_regex]],
                                     [name: ["$not": migration_regex]]]]
 
-  defp list_collections(repo, opts) do
+  def list_collections(_ = "3.2", repo, opts) do
+    colls = command(repo, %{"listCollections": 1}, opts)
+
+    colls["cursor"]["firstBatch"] |> Enum.map(&Map.fetch!(&1, "name"))
```

Comment:
Please don't use pipe with just one element. 

We also need to exclude the `schema_migrations` table.